sdk/tables/azure-data-tables/azure/data/tables/_error.py

```diff
@@ -215,6 +205,27 @@ def _process_table_error(storage_error, table_name=None):
     _reraise_error(decoded_error)
 
 
+def _reprocess_error(decoded_error, identifiers=None):
+    error_code = decoded_error.error_code
+    message = decoded_error.message
+    invalid_input = "The number of keys specified in the URI does not match number of key properties for the resource"
+    invalid_query_parameter_value = "Value for one of the query parameters specified in the request URI is invalid"
+    properties_need_value = "The values are not specified for all properties in the entity"
+    table_does_not_exist = "The table specified does not exist"
+    if (error_code == "InvalidInput" and invalid_input in message or # pylint: disable=too-many-boolean-expressions
+        error_code == "InvalidQueryParameterValue" and invalid_query_parameter_value in message or
+        error_code == "PropertiesNeedValue" and properties_need_value in message or
+        error_code =="TableNotFound" and table_does_not_exist in message
+        ):
+        raise ValueError(message + "\nNote: Try to remove the table name in the end of endpoint if it has.")
```

Comment:
```python
decoded_error.message = decoded_error.message + "\nA possible cause of this error could be that the account URL used to create the Client includes an invalid path, for example the table name. Please check your account URL."
raise decoded_error
```

Reply:
The output is not from `decoded_errormessage` by my tests, I changed it to update by `decodec_error.args`.